hudi-support/extensions/pom.xml

```diff
@@ -92,6 +106,14 @@
             <groupId>com.esotericsoftware</groupId>
             <artifactId>kryo</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
```

Comment:
can & should these be provided ?

Reply:
Yes and they are set as provided already at the project level